File: Exercise/Medium/11. Count Occurrences in Array/Code/Count Occurrences in Array.cs
using System;

class Program {
    static void ReadVector(int[] numbers) {
        string line = Console.ReadLine();
        string[] tokens = line.Split(' ');

        for (int i = 0; i < numbers.Length; i++) {
            numbers[i] = int.Parse(tokens[i]);
        }
    }

    static void FindNumber(int[] numbers, int num) {
        int counter = 0;
        foreach (int number in numbers) {
            if (number == num) {
                counter++;
            }
        }
        Console.WriteLine(counter);
    }

    static void Main() {
        int n = int.Parse(Console.ReadLine());
        int[] numbers = new int[n];

        ReadVector(numbers);

        int num = int.Parse(Console.ReadLine());

        FindNumber(numbers, num);
    }
}

File: Exercise/Medium/11. Count Occurrences in Array/Code/Count Occurrences in Array.py
import sys

def read_vector(numbers):
    line = sys.stdin.readline().strip()
    istr = line.split()

    for num in istr:
        numbers.append(int(num))

def find_number(numbers):
    num = int(input())

    counter = 0
    while numbers:
        if numbers.pop() == num:
            counter += 1

    print(counter)

if __name__ == "__main__":
    numbers = []

    read_vector(numbers)
    find_number(numbers)